src/sentry/replays/blueprints/api.md

```diff
@@ -64,6 +64,7 @@ This document is structured by resource with each resource having actions that c
     | replay_click.testid      | string        | The data-testid of an HTML element. (omitted from public docs) |
     | replay_click.textContent | string        | The text-content of an HTML element.                           |
     | replay_click.title       | string        | The title attribute of an HTML element.                        |
+    | replay_click.selector    | string        | A valid CSS selector.                                          |
```

Comment:
Super late in the game for this suggestion but I'll do it anyway: Could we name this `click.selector`?

Reply:
@bruno-garcia Let's handle this in a follow-up PR.  We need to ask @ryan953 his reasoning when he returns from vacation.